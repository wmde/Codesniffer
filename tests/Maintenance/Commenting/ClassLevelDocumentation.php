<?php

/**
 * This should not be reported as an error, but everything below should.
 */
class MyCompliantClass {
}

/**
 * Missing newline between comment and class.
 */class MissingNewline {
}

/**
 * Wrong whitespace character between comment and class.
 */ class WrongWhitespace {
}

/**
 * To much whitespace between comment and class.
 */

class ToMuchWhiteSpace {
}

/*
 * This is not a PHPDoc comment.
 */
class NotAPhpDocComment {
}

interface UndocumentedInterface {
}

class UndocumentedClass {
}

trait UndocumentedTrait {
}

class UndocumentedImplementation implements UndocumentedInterface {
}

class UndocumentedSubClass extends UndocumentedClass {
}

namespace UndocumentedNamespace {
	class UndocumentedClassInNamespace {
	}
}
