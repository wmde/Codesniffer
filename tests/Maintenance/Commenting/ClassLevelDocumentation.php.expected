 11 | WARNING | Unexpected whitespace after class level documentation
 16 | WARNING | Unexpected whitespace after class level documentation
 22 | WARNING | Unexpected whitespace after class level documentation
 28 | ERROR   | Regular comment found instead of class level documentation
 32 | ERROR   | Class level documentation missing
 35 | ERROR   | Class level documentation missing
 38 | ERROR   | Class level documentation missing
 41 | ERROR   | Class level documentation missing
 44 | ERROR   | Class level documentation missing
 48 | ERROR   | Class level documentation missing
